editoast/src/schema/v2/trainschedule.rs

```diff
@@ -35,6 +36,26 @@ pub struct TrainScheduleBase {
     pub options: TrainScheduleOptions,
 }
 
+impl From<TrainSchedule> for TrainScheduleBase {
```

Comment:
This shouldn't be done here cause:
- We are trying to keep the From impl in views
- We will soon split editoast in multiple crates. Two of which will be `Schema` and `model`, and the model crate will depend on the schema. 

-------

What can you do:
- Move this in the `modelv2::train_schedule` module
- Remove this impl and use `TrainScheduleResult`. So you can do: `ts.into::<TrainScheduleResult>().train_schedule`